### DISCUSSION TITLE
Click/touch for popover

### DISCUSSION THREAD

**Discussion by eattiyeh** [MMRIZE/MMM-CalendarExt3#182] [Q&A]
I am running MM server on an RPi and viewing via Chrome on the client. Although I can get other modules (eg, MMM-SmartTouch) to respond to mouse clicks or screen touches, I cannot get event popovers to work in CX3. Even with 'showMore' enabled, nothing happens when I click/touch the overflow indicator number. Is there a setting I am missing to enable this? Or is it not possible with a client/server setup? 

Thanks.

**Comment by eouia** [comment]
Without MMM-SmartTouch, still not work?
I suspect MMM-SmartTouch might cover the whole viewpoint to get a touch from the screen, then not to deliver the event.(propergation)
So my module cannot catch the touch event.

**Reply by eattiyeh** [reply]
The issue existed before I installed MMM-SmartTouch. I added MMM-SmartTouch to make sure I could get anything to respond to a click or touch.

I tried hiding every module except for CX3, but still no response to a click.

Appreciate any suggestions. Thanks!

**Comment by eouia** [comment]
It was my fault. I made some mistakes in the previous version. I fixed it.

**Reply by eattiyeh** [reply]
Thanks!! Works now for the showMore numerical indicator and when clicking on the day of the week to show all of the day's events in the popover.

But it still does not work for individual events to show a popover with the event details (which is a great feature that adds a lot to this calendar's functionality). Happy to provide any other details needed for debugging.

**Reply by eouia** [reply]
AH... it was my mistake also. I was confused that it was an unintended bug, so I disabled that feature in this fix. (I forgot the reason for some codes.) I'll look inside and fix it as soon as possible.

**Reply by roysim11** [reply]
Just wanted to add something here, popovers work great for me for both the day's total events and then the individual events.  However, the date formats are different.  The day's total events display correctly, which would be en-US (mm/dd/yr) 12h AM/PM.  The format for the individual popovers is (dd/mm/yr) 24hr.  Is there a config line to get the individual event format to match the total events format?
